.github/pull_request_template.md

```diff
@@ -56,6 +56,8 @@ _[Add and remove items below accordingly]_
 - [ ] Upstream dependencies are met/released
 - [ ] Release NPM packages
 - [ ] Fern Definition Updated
+- [ ] Sandbox, Staging, Production FFs have been set.
+- [ ] No dependencies between API and Infra that will cause errors during deployment
```

Comment:
nits:
- keep Fern related together (definition and SDKs)
- FFs: start the line with the main concept (helps w/ scanning), like "FFs have been set in Staging, Production, and Sandbox"